clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -937,13 +937,53 @@ check_schedule_file()
     return true;
```

Comment:
> …ncrease.

Consider shrinking the PR title so it doesn't wrap here and in some other tools: maybe end with "... that timestamps never decrease" or sthg.

Reply:
Done. I've changed it to "i#6187: Add invariant check that timestamps never decrease"